CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+### 2.21.2
+This release includes:
+* An Amazon Linux 2 Base
+* Fluent Bit [1.8.9](https://fluentbit.io/announcements/v1.8.9/)
+* Amazon CloudWatch Logs for Fluent Bit 1.6.3
+* Amazon Kinesis Streams for Fluent Bit 1.8.0
+* Amazon Kinesis Firehose for Fluent Bit 1.6.1
+
+Compared to `2.21.1` this release includes the following fixes for AWS customers that we are working on getting accepted upstream:
+* Bug - Resolve IMDSv1 fallback error introduced in 2.21.0 [aws-for-fluent-bit:259](https://github.com/aws/aws-for-fluent-bit/issues/259)
+
```

Comment:
Need to also add the CW go plugin change right?

Reply:
Done

Reply:
Now the "following fixes for AWS customers that we are working on getting accepted upstream" language is technically inaccurate. its only true for one of them